### PR TITLE
3.0.0

### PR DESCRIPTION
**Pull request by stephenwright**
Updated signature of siren helper methods.
Updated README.
Changed the way `select` works.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.1.0",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-import": "^2.20.2",
```
Comment (ThomasMorrison) [inline package.json:27]: Did you intentionally switch from yarn/yarn.lock -> npm/package-lock.json?

File: README.md [hunk 2]
```diff
@@ -9,3 +9,61 @@ Enter an API endpoint to render all the entity's properties, links, actions, and
...
+      "title": "Disable",
+      "value": false
+    }
+  ]
+}
+```
+
+### The "checked" field property
+
+Shipwreck supports the field having a `checked` property for inputs of type `radio` and `checkbow`
```
Comment (ThomasMorrison) [inline README.md:46]: checkbow => checkbox

File: README.md [hunk 3]
```diff
@@ -9,3 +9,61 @@ Enter an API endpoint to render all the entity's properties, links, actions, and
...
+
+### The "select" field type
+
+`select` is not valid field type in the siren spec. Shipwreck supports `select`, and expects `value` to be an array.
```
Comment (ThomasMorrison) [inline README.md:50]: is not valid field... => is not a valid field...

File: package.json (hunk 1 above)
Comment (stephenwright) [reply package.json:27]: yes. i thought for a public project sticking to the default was better than picking favourites.

**Pull request merged**